Code/Framework/AzCore/Platform/Common/WinAPI/AzCore/Socket/AzSocket_WinAPI.h

```diff
@@ -8,7 +8,7 @@
  */
```

Comment:
Is this file even used anymore after your changes? All our socket impl's should live in AzNetworking now and shouldn't have any aspects laying around in AzCore.

Reply:
That's a good question. This got caught in my cleanup pass so I'd have to double check.